demos/bridge/primefaces3-portlet/src/main/webapp/views/applicant.xhtml

```diff
@@ -66,7 +66,7 @@
 									<f:selectItem itemLabel="#{i18n['select']}" itemValue="" />
 									<f:selectItems itemValue="#{province.provinceId}" itemLabel="#{province.provinceName}"
 										value="#{listModelBean.provinces}" var="province" />
-									<p:ajax update="provinceIdField" />
+									<f:ajax render="provinceIdField" />
 								</h:selectOneMenu>
```

Comment:
@stiemannkj1 Forgot to mention please do this in primefaces3-portlet, primefaces4-portlet, and primefaces5-portlet.